### PR TITLE
Remove topology if test does not require

### PR DESCRIPTION
**Pull request by chloerh**
Since test could hit BZ1426220 if topology not removed.

Signed-off-by: haizhao <haizhao@redhat.com>

### CODE REVIEW

**Comment by chloerh** [comment]
Requires @dzhengfy 's review

File: libvirt/tests/src/virsh_cmd/domain/virsh_setvcpus.py [hunk 1]
```diff
@@ -163,6 +163,13 @@ def set_expected(vm, options):
             vmxml['cpu'] = vmcpu_xml
             vmxml.sync()
 
+        # Remove topology if test does not requires
```
Comment (dzhengfy) [inline libvirt/tests/src/virsh_cmd/domain/virsh_setvcpus.py:166]: I saw bz1426220 is only PPC bug. So I think we'd better not to make the degradation for x86_64 or other arches. Please limit this change too PPC only.

Comment (chloerh) [reply libvirt/tests/src/virsh_cmd/domain/virsh_setvcpus.py:166]: I think it's a coincidence that this bug I found was reported on ppc. Then I found another bug related to this issue, bz1370066.

Comment (dzhengfy) [reply libvirt/tests/src/virsh_cmd/domain/virsh_setvcpus.py:166]: OK. As this is also an issue on x86_64, I agree to remove it here.

**Pull request merged**